### ISSUE TITLE
Notifications are re-enabled after reloading

### ISSUE THREAD

**Issue by FWDekker** [getferdi/ferdi#257]
**Describe the bug**
If notifications are disabled, they will be re-enabled after reloading Ferdi or restarting Ferdi.

**To Reproduce**

1. Disable notifications (<kbd>Ctrl + Shift + M</kbd>).
2. Reload (<kbd>Ctrl + Shift + R</kbd>) or restart Ferdi.
3. Wait 5 seconds.

After reloading, the notification icon will still be disabled. However, after 5 seconds, notifications will be enabled again.

**Expected behavior**
Notifications remain disabled after reloading or restarting Ferdi.

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Ubuntu 18.04
- Browser [e.g. chrome, safari]: Electron 6.0.10
- Version [e.g. 22]: 5.4.0

**Comment by kytwb** [comment]
I guess this can be explained with the notifications mute button state not being saved as part of the user settings. Doing so should fix this issue.

**Comment by reischlfranz** [comment]
I have the inverse problem - I would want the notifications on, but reload / restart disables them again. This is a problem that already persisted in Franz for over a year now.

Interesting however that for some it starts on, for others at off.
* Win10 Pro x64 
* Version 5.4.3

**Comment by vantezzen** [comment]
Sorry if I am the only one misunderstanding this feature but isn't it supposed to be this way?
I always thought of this option as a temporary one, e.g. to temporarily mute services and audio while you are on a call and that it will auto-reset after restarting Ferdi as to not confuse users why the audio/notifications isn't working. The permanent way to disable notifications and/or audio in this case would be to disable it in the service settings.

**Comment by kytwb** [comment]
I'd tend to side with @vantezzen on this one; if you want to permanently disable audio/notifications for a service, you can do it in the service settings and those should be saved:

<img width="449" alt="Screen Shot 2020-02-06 at 12 12 45 PM" src="https://user-images.githubusercontent.com/412895/74196060-c2f8aa00-4c5c-11ea-92b6-1fea06fde7f8.png">

**Comment by FWDekker** [comment]
Fair enough. In that case I think it should be made clearer that notifications are only disabled until Ferdi is restarted.

**Comment by reischlfranz** [comment]
> @reischlfranz can you check that you have the above enabled in your service settings?
Yes.

I would be fine with the function being "temporarily disable", but that just kicks the can down the road. Since no global setting exists to set the default behavior to notifications on or off. And, since in my case, the notifications start as disabled by default on every launch or reload, I only have a "temporarily **enable**" button there.

Which is why I feel a persistent global setting for notification settings is required in some form - be it the bell button, or a line in the options menu.

**Comment by FWDekker** [comment]
Not stale.

**Comment by Suzaru** [comment]
Not stale
